test/e2e-common.sh

```diff
@@ -58,12 +55,79 @@ function tektonconfig_ready_wait() {
     echo waiting for TektonConfig config Ready status
     sleep 5
     kubectl get TektonConfig config > /dev/null 2>&1
```

Comment:
can we remove this as we are checking it in `is_tektonconfig_cr_created`

Reply:
done